graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -134,6 +134,16 @@ def test_graph_drop(gds: GraphDataScience) -> None:
         gds.graph.drop(G, True)
 
 
+def test_graph_drop_by_name(gds: GraphDataScience) -> None:
+    gds.graph.project(GRAPH_NAME, "*", "*")
+
+    result = gds.graph.drop_by_name(GRAPH_NAME, True)
```

Comment:
```suggestion
    result = gds.graph.get(GRAPH_NAME).drop(True)
```

I guess I don't see why the `drop_by_name` is better than `get().drop()`
If the problem is that `get()` will report an error if it does not find anything, I would be open to adding a boolean `failIfMissing` also to `get()`.

Reply:
@Mats-SX yes, `get()` returning an error when not found would be the issue. I agree adding a `failIfMissing` parameter is a more elegant solution to this problem.